jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/MBeanHelper.groovy

```diff
@@ -123,7 +123,16 @@ class MBeanHelper {
 
     Object getBeanAttributeWithTransform(GroovyMBean bean, String attribute){
       def transformationClosure = attributeTransformation.get(attribute);
-      return transformationClosure != null ? transformationClosure(bean) : getBeanAttribute(bean, attribute)
+      if (transformationClosure != null){
```

Comment:
nit: I like inverting this and returning `null` early because it helps to reduce the indent levels (reader burden).